addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1,4 +1,7 @@
 # Kodi Media Center language file
+# [Capitalization] Avoid capitalizing every second word e.g. "This New String", unless proper name or proper noun.
```

Comment:
give an example of what and what not
"This new string" instead of "This New String"
Use more lines because text now wraps. Also make clear that devs need to look if an acceptable string already exists.
Must do's:
Add description of the string using   "#. This describes a certain setting"
Add location where string is used   "#: xbmc/addons/guidialogaddoninfo.cpp"